aspnetcore/includes/blazor-preview-notice.md

```diff
@@ -0,0 +1,6 @@
+> [!WARNING]
+> **Blazor is an unsupported experimental web framework. Blazor is in preview.**
+>
+> Blazor preview documentation is provided to assist you in trying out Blazor. Blazor isn't a committed Microsoft product and isn't supported by a Microsoft SLA at this time. Blazor's feature set, capabilities, and tooling are subject to change at any time without notice. Blazor shouldn't be used for production workloads. Microsoft makes no warranties, express or implied, with respect to the information provided in this preview documentation or for the Blazor technology itself.
```

Comment:
I think you can remove the latest sentence.

I would also replace the 2nd sentence with: "Blazor isn't a committed product at this time."